c10/core/SymInt.h

```diff
@@ -93,7 +93,7 @@ class C10_API SymInt {
     // https://stackoverflow.com/questions/42534749/signed-extension-from-24-bit-to-32-bit-in-c
     uint64_t extended_bits = (unextended_bits ^ sign_bit_mask) - sign_bit_mask;
     return static_cast<SymNodeImpl*>(
-        // NOLINTNEXTLINE(performance-no-int-to-ptr)
+        // NOLINTNEXTLINE
```

Comment:
Why? Specific nolintnextlines are good (and you can specify multiple of them if needed).

Reply:
There is another one to suppress that makes the line to long to format...

Reply:
Can you glob them together at least?

Reply:
Fixed.